hosted_control_planes/hcp-deploy-disconnected.adoc

```diff
@@ -4,4 +4,106 @@ include::_attributes/common-attributes.adoc[]
+
+* For instructions to customize ingress and DNS behavior for a virtual environment, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-virt.adoc#hcp-virt-ingress-dns-custom[Customizing ingress and DNS behavior].
+
+include::modules/hcp-dc-registry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-virt.adoc#hcp-dc-mgmt-cluster_hcp-deploy-virt[Setting up a management cluster for {hcp} in a disconnected environment]
+
+include::modules/hcp-dc-web-server.adoc[leveloffset=+1]
+include::modules/hcp-dc-image-mirror.adoc[leveloffset=+1]
+include::modules/hcp-dc-apply-objects.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* If you are working in a virtual environment, after you configure mirroring, ensure that you meet the xref:../hosted_control_planes/hcp-deploy/hcp-deploy-virt.adoc[prerequisites for hosted control planes on OpenShift Virtualization].
+
+* For more information about mirroring nightly or CI versions of {product-title}, see xref:../installing/disconnected_install/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[Mirroring images for a disconnected installation using the oc-mirror plugin].
+
+[id="hcp-dc-mce"]
+== Deploying {mce-short} for a disconnected installation of {hcp}
+
+The {mce} plays a crucial role in deploying clusters across providers. If you do not have {mce-short} installed, review the following documentation to understand the prerequisites and steps to install it:
+
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#mce-intro[About cluster lifecycle with multicluster engine operator]
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#mce-install-intro[Installing and upgrading multicluster engine operator]
+
+include::modules/hcp-agentserviceconfig.adoc[leveloffset=+2]
+
+include::modules/hcp-dc-tls.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../security/certificates/updating-ca-bundle.adoc#updating-ca-bundle[Updating the CA bundle]
+
+[id="hcp-dc-hosted"]
+== Deploying the hosted cluster in a disconnected environment
+
+A hosted cluster is an {product-title} cluster with its control plane and API endpoint hosted on a management cluster. The hosted cluster includes the control plane and its corresponding data plane.
+
+The following procedures use manifests, which provide flexibility for modifying the related artifacts.
+
+include::modules/hcp-hc-objects.adoc[leveloffset=+2]
+include::modules/hcp-nodepool-hc.adoc[leveloffset=+2]
+include::modules/hcp-dc-infraenv.adoc[leveloffset=+2]
+include::modules/hcp-worker-hc.adoc[leveloffset=+2]
+include::modules/hcp-bm-hosts.adoc[leveloffset=+2]
+include::modules/hcp-dc-scale-np.adoc[leveloffset=+2]
```

Comment:
bare metal specific

Reply:
Noted. Moved to a bare-metal-specific section of the disconnected docs.